### PR TITLE
Update the readme

### PR DESCRIPTION
**Pull request by Luyunmt**
Update the readme to point to the samples in the mono-repo that do the same thing as what is in this repo.

### CODE REVIEW

**Comment by jongio** [comment]
@conniey - Can you review?

File: README.md [hunk 1]
```diff
@@ -49,6 +47,20 @@ To run this sample:
...
+The key vault SDK package  here is **com.microsoft.azure.azure-keyvault**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please reference to the following examples:
+
+* [IdentityReadmeSamples.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/IdentityReadmeSamples.java)- Examples to authenticate to key vault secret client
+
+    * createClientCertificateCredential: Create a secret client using cert based authentication.
+    * createDefaultAzureCredential: Create a secret client using DefaultAzureCredential.
+
+* [HelloWorld.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/HelloWorld.java)  - Examples for common key vault tasks:
+
+    * Create a secret inside the keyvault
```
Comment (conniey) [inline README.md:61]: keyvault -> Key Vault

Same with other usages of "key vault". It's a product name.

File: README.md [hunk 2]
```diff
@@ -49,6 +47,20 @@ To run this sample:
 6. Run the sample:
     - mvn clean compile exec:java
 
+## Use latest key vault SDK
+
+The key vault SDK package  here is **com.microsoft.azure.azure-keyvault**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please reference to the following examples:
```
Comment (conniey) [inline README.md:52]: IIRC you have another PR that uses the same Key Vault content. My comments still apply. (I cannot find that PR at the moment.)

File: README.md [hunk 3]
```diff
@@ -49,6 +47,17 @@ To run this sample:
 6. Run the sample:
     - mvn clean compile exec:java
 
+## Use latest Key Vault SDK
+
+The Key Vault secrets SDK here is **com.azure.azure-security-keyvault-secrets**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please refer to the following examples:
```
Comment (conniey) [inline README.md:52]:
```suggestion
The Key Vault secrets SDK here is **com.azure:azure-security-keyvault-secrets**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please refer to the following examples:
```

File: README.md [hunk 4]
```diff
@@ -49,6 +47,17 @@ To run this sample:
...
+
+* [IdentityReadmeSamples.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/IdentityReadmeSamples.java) shows multiple ways to authenticate the Key Vault client via DefaultAzureCredential, device code, client secret or certificate in addition to others.
+
+* [HelloWorld.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/HelloWorld.java)  - Examples for common Key Vault tasks:
```
Comment (conniey) [inline README.md:56]:
```suggestion
* [HelloWorld.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/HelloWorld.java) - Examples for common Key Vault tasks:
```

File: README.md [hunk 5]
```diff
@@ -49,6 +47,17 @@ To run this sample:
...
+
+    * Create a secret inside the Key Vault
+    * Get the secret
+
 ## More information
 
 [http://azure.com/java](http://azure.com/java)
```
Comment (conniey) [inline README.md:63]:
```suggestion
https://azure.com/java
```

Comment (conniey) [reply README.md:63]: Markdown by default understands links, you don't need to specify it. Also use https by default.

**Review by conniey — APPROVED** [review]
Some suggestions

**Pull request merged**